shopfloor_mobile/static/wms/src/components/screen.js

```diff
@@ -59,7 +59,8 @@ Vue.component("Screen", {
                 this.$root.authenticated &&
                 !this.$root.has_profile &&
                 this.$route.name != "profile" &&
-                this.$route.name != "settings"
+                this.$route.name != "settings" &&
+                this.$route.name != "workstation"
```

Comment:
pls add a TODO that reminds to add a metadata flag to routes (eg: `profile_required`) then we can remove this hardcoded check.

Reply:
Added just above